packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -493,6 +493,37 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 /// [ImageFiltered] instead. For that scenario, [ImageFiltered] is both
 /// easier to use and less expensive than [BackdropFilter].
 ///
+/// {@tool snippet}
+///
+/// This example shows how the common case of applying a [BackdropFilter] blur
+/// to a single child can be replaced with an [ImageFiltered] widget. This code
+/// is generally simpler and the performance will be improved dramatically.
+///
+/// ```dart
+///  return Stack(
+///    children: [
+///      Positioned.fill(child: Image.asset('image.png')),
+///        Positioned.fill(
+///          child: BackdropFilter(
+///            filter: ImageFilter.blur(sigmaX: 6, sigmaY: 6),
+///          ),
+///        ),
+///      ],
+///   );
+///
+/// ```
+///
+/// Instead consider the following:
```

Comment:
I believe this will not render correctly in dart doc as dartdoc will pull all the regular text up front before any code blocks.

Reply:
Should I just do two separate tool blocks then?